src/systemctl/systemctl-switch-root.c

```diff
@@ -37,7 +38,9 @@ int verb_switch_root(int argc, char *argv[], void *userdata) {
         }
 
         init = empty_to_null(init);
-        if (init) {
+        if (getenv_bool("SYSTEMCTL_FORCE_HANDOVER") > 0)
+            init = NULL;
```

Comment:
indentation off.

But I really don't get this. You already can specify an empty string as third argument to achieve exactly this behaviour here. i.e. the `empty_to_null()` makes sure that an empty string is automatically turned into `NULL`.

i.e. if you invoke this:

```sh
systemctl switch-root /sysroot ""
```

you should already get exactly what you want? what am i missing?

Reply:
Thanks, that works like I need it :) systemd already doing what I need without me patching it again :P